tests/test_shutdown.py

```diff
@@ -1,2 +1,85 @@
 # TODO: Add tests that show proper operation of this strategy through "emergencyExit"
 #       Make sure to demonstrate the "worst case losses" as well as the time it takes
+
+import brownie
+import pytest
+
+AddressZero = "0x0000000000000000000000000000000000000000"
+
+def test_emergency_permissions_deny(strategy, strategist, gov, guardian, management, accounts):
+  with brownie.reverts("!authorized"):
+    strategy.setEmergencyExit({"from": accounts[9]})
+
+def test_emergency_permissions_strategist(strategy, strategist, gov, guardian, management, accounts):
+    strategy.setEmergencyExit({"from": strategist})
+    assert strategy.emergencyExit() == True
+
+def test_emergency_permissions_gov(strategy, strategist, gov, guardian, management, accounts):
+    strategy.setEmergencyExit({"from": gov})
+    assert strategy.emergencyExit() == True
+
+def test_emergency_permissions_guardian(strategy, strategist, gov, guardian, management, accounts):
+    strategy.setEmergencyExit({"from": guardian})
+    assert strategy.emergencyExit() == True
+
+def test_emergency_permissions_management(strategy, strategist, gov, guardian, management, accounts):
+    strategy.setEmergencyExit({"from": management})
+    assert strategy.emergencyExit() == True
```

Comment:
not sure why you want these. Those tests exist at the yearn-vaults repo.

Reply:
Fair, removed them